change/@ni-nimble-components-15e8f878-a870-4090-8bf6-7c45c65195ba.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I saw this warning in the debug console when I loaded the [table demo page](https://60e89457a987cf003efc0a5b-wjfhtkxgzq.chromatic.com/example-client-app/#/tabledemo) in Firefox. I didn't see the same warning on the main example page in main or in your branch. Thoughts on whether it's relevant?

"This site appears to use a scroll-linked positioning effect. This may not work well with asynchronous panning; see https://firefox-source-docs.mozilla.org/performance/scroll-linked_effects.html for further details and to join the discussion on related tools and features!"

Reply:
Yeah Firefox prints out that message whenever the virtualized table is scrolled (you'll see it [in Storybook with the large data set](https://60e89457a987cf003efc0a5b-wjfhtkxgzq.chromatic.com/?path=/story/table--table&args=data:LargeDataSet) too, once you scroll down in the table).

It does seem to be new with the implementation changes in this PR (i.e. with the 2nd div - the [prototype Storybook](https://60e89457a987cf003efc0a5b-haosfwmjoq.chromatic.com/iframe.html?args=&id=table--table-story&viewMode=story) doesn't print the same message).  

But I don't think it's a concern because the Firefox rendering performance is fine with this approach (and better than it was with the single div / many-translated-rows approach).